Week1/homework.js

```diff
@@ -0,0 +1,253 @@
+let mysql = require('mysql');
+
+// establish db
+let db = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'world',
+});
+
+// connect database
+db.connect(err => {
+  if (err) throw err;
+  console.log('Mysql connected....');
+});
+
+// drop database
+let drop_db = `DROP DATABASE IF EXISTS world`;
```

Comment:
You should leave this part out if you want to use world.sql for loading data because if you run world.sql before running this, this command would remove the database.